sigle/src/modules/analytics/PublishedStoriesFrame.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import { SubsetStory } from '../../types';
+import { Box } from '../../ui';
+import { Pagination } from './Pagination';
+import { PublishedStoryItem } from './PublishedStoryItem';
+
+interface PublishedStoriesFrameProps {
+  stories: SubsetStory[];
+}
+
+export const PublishedStoriesFrame = ({
+  stories,
+}: PublishedStoriesFrameProps) => {
+  const [currentStories, setCurrentStories] = useState<SubsetStory[] | null>(
+    stories
+  );
+  const [currentPage, setCurrentPage] = useState<number>(1);
+
+  // how many stories we should show on each page
+  let itemSize = 7;
+
+  useEffect(() => {
+    const firstPageIndex = (currentPage - 1) * itemSize;
+    const lastPageIndex = firstPageIndex + itemSize;
+    setCurrentStories(stories.slice(firstPageIndex, lastPageIndex));
+  }, [currentPage]);
+
+  return (
+    <>
+      {currentStories && (
+        <>
+          <Box css={{ mb: '$3', height: 476 }}>
```

Comment:
fixed height to prevent pagination component shifting if the next or previous page has more or less items